appengine/bigquery/requirements.txt

```diff
@@ -1 +1 @@
-google-api-python-client
+google-api-python-client==1.4.2
```

Comment:
Yeah, in this case I just want to have the confidence that what we're giving to our users works.